openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Services.ts

```diff
@@ -0,0 +1,206 @@
+    .contains(services)
+    .click();
+
+  verifyResponseStatusCode('@getServiceList', 200);
+};
+
+export const getEntityTypeFromService = (service: Services) => {
+  switch (service) {
+    case Services.Dashboard:
+      return EntityType.DashboardService;
+    case Services.Database:
+      return EntityType.DatabaseService;
+    case Services.Storage:
+      return EntityType.StorageService;
+    case Services.Messaging:
+      return EntityType.MessagingService;
+    case Services.Search:
+      return EntityType.SearchService;
+    case Services.MLModels:
+      return EntityType.MlModelService;
+    case Services.Pipeline:
+      return EntityType.PipelineService;
+    default:
+      return EntityType.DatabaseService;
+  }
+};
+
+export const retryIngestionRun = () => {
+  interceptURL('GET', '/api/v1/services/*/name/*', 'serviceDetails');
+  interceptURL(
+    'GET',
+    '/api/v1/services/ingestionPipelines/*/pipelineStatus/*',
+    'pipelineStatus'
+  );
+  let timer = BASE_WAIT_TIME;
+  let retryCount = 0;
+  const testIngestionsTab = () => {
+    cy.get('[data-testid="ingestions"]').scrollIntoView().should('be.visible');
+    cy.get('[data-testid="ingestions"] >> [data-testid="count"]').should(
+      'have.text',
+      '1'
+    );
+    if (retryCount === 0) {
+      cy.wait(1000);
+      cy.get('[data-testid="ingestions"]').should('be.visible');
+    }
+  };
+
+  const checkSuccessState = () => {
+    testIngestionsTab();
+    retryCount++;
+
+    // the latest run should be success
+    cy.get('[data-testid="pipeline-status"]').then(($ingestionStatus) => {
+      if ($ingestionStatus.text() !== 'Success' && retryCount <= RETRY_TIMES) {
+        // retry after waiting with log1 method [20s,40s,80s,160s,320s]
+        cy.wait(timer);
+        timer *= 2;
+        cy.reload();
+        verifyResponseStatusCode('@serviceDetails', 200);
+        verifyResponseStatusCode('@pipelineStatus', 200);
+        checkSuccessState();
+      } else {
+        cy.get('[data-testid="pipeline-status"]').should('contain', 'Success');
+      }
+    });
+  };
+
+  checkSuccessState();
+};
+
+export const deleteService = (typeOfService: Services, serviceName: string) => {
+  interceptURL(
+    'GET',
+    'api/v1/search/query?q=*&from=0&size=15&index=*',
+    'searchService'
+  );
+  cy.get('[data-testid="searchbar"]').type(serviceName);
+
+  verifyResponseStatusCode('@searchService', 200);
+
+  // click on created service
+  cy.get(`[data-testid="service-name-${serviceName}"]`).click();
+
+  cy.get(`[data-testid="entity-header-display-name"]`)
+    .invoke('text')
+    .then((text) => {
+      expect(text).to.equal(serviceName);
+    });
+
+  // Clicking on permanent delete radio button and checking the service name
+  cy.get('[data-testid="manage-button"]').click();
+
+  cy.get('[data-menu-id*="delete-button"]').should('be.visible');
+  cy.get('[data-testid="delete-button-title"]').click().as('deleteBtn');
```

Comment:
```suggestion
  cy.get('[data-testid="delete-button-title"]').click();
```
as we are not using it